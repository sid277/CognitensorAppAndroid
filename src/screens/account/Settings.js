import React, { useState } from 'react';
import { Text, Switch, StyleSheet, View } from 'react-native';
import DefaultView from '../../components/default/DefaultView';
import { theme } from '../../theme';

const Settings = () => {
  const [notification, setNotification] = useState(false);

  const toggleNotification = () => {
    setNotification(!notification);
  };

  return (
    <DefaultView>
      <View style={styles.item}>
        <Text style={styles.itemText}>Notifications</Text>
        <Switch value={notification} onValueChange={toggleNotification} />
      </View>
    </DefaultView>
  );
};

const styles = StyleSheet.create({
  item: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignSelf: 'stretch',
    alignItems: 'center',
    backgroundColor: '#FBFBFB',
    padding: theme.spacing.base,
  },
  itemText: {
    ...theme.typography.body,
  },
});

export default Settings;
