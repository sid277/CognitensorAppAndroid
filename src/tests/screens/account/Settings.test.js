import React from 'react';
import { shallow } from 'enzyme';
import Settings from '../../../screens/account/Settings';
import renderer from 'react-test-renderer';


test('should test Settings component', () => {
const wrapper = shallow(<Settings />);
expect(wrapper).toMatchSnapshot();
});