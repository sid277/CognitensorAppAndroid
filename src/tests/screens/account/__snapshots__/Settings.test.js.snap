// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Settings component 1`] = `ShallowWrapper {}`;
